libs/cardano-ledger-core/cardano-ledger-core.cabal

```diff
@@ -113,7 +113,7 @@ library
         measures,
         microlens,
         network,
-        nothunks,
+        nothunks ^>=0.1.5,
```

Comment:
Version after this has an API change. Tried to add a comment but then `cabal-format` complained.

Reply:
Could you please post here the build error that we get when using `nothunks-0.2.x` (i.e. without this constraint)?

If we can't accommodate it in this PR then we'll be able to turn this comment into its own issue and take care of it later.

Reply:
https://github.com/IntersectMBO/cardano-ledger/issues/4504